vscode/src/debugger/activate.ts

```diff
@@ -113,19 +113,7 @@ class QsDebugConfigProvider implements vscode.DebugConfigurationProvider {
     config: vscode.DebugConfiguration,
     _token?: vscode.CancellationToken | undefined,
   ): vscode.ProviderResult<vscode.DebugConfiguration> {
-    // if launch.json is missing or empty
-    if (!config.type && !config.request && !config.name) {
-      const docUri = getActiveQSharpDocumentUri();
-      if (docUri) {
-        config.type = "qsharp";
-        config.name = "Launch";
-        config.request = "launch";
-        config.programUri = docUri.toString();
-        config.shots = 1;
-        config.noDebug = "noDebug" in config ? config.noDebug : false;
-        config.stopOnEntry = !config.noDebug;
-      }
-    } else if (config.program && folder) {
+    if (config.program && folder) {
       // A program is specified in launch.json.
```

Comment:
Would be good to test with a `launch.json` present, and also maybe have @idavis give the thumbs up before merging.

Reply:
Will do!

Reply:
Confirmed it behaves as expected with a launch.json (uses the file specified in that configuration, along with relevant settings).